scripts/start_internal.sh

```diff
@@ -0,0 +1,114 @@
+#!/bin/bash
```

Comment:
same comment here about doing a final diff with the original file before merging

Reply:
Done.

https://gist.githubusercontent.com/trironkk/feb822f407ac3d534806aef128c81b97/raw/cf0ea9ddae6157c9b342fed55eb32f6828308cc0/gistfile1.txt